apps/jetbrains.py

```diff
@@ -510,12 +512,20 @@ def build_text_action_keymap():
         "extract parameter": idea("action IntroduceParameter"),
         "extract interface": idea("action ExtractInterface"),
         "extract method": idea("action ExtractMethod"),
+        "refactor in line": idea("action Inline"),
+        "refactor move": idea("action Move"),
+        "refactor rename": idea("action RenameElement"),
+        "rename file": idea("action RenameFile"),
+        "(organize | optimize) imports": idea("action OptimizeImports"),
         # Quick Fix / Intentions
         "fix this <dgndictation> [over]": [
             idea("action ShowIntentionActions"),
             utils.text,
         ],
         "fix (format | formatting)": idea("action ReformatCode"),
+        # indentation
+        "indent": idea("action EditorIndentLineOrSelection"),
```

Comment:
tab / shift-tab covers these.  I'm unsure if we want collisions with textwrap.dedent from python, as well.

Reply:
actually meant to put these into selection verbs as well. i use it when im in the middle of a line. in that case tab actually inserts a tab in the middle of the line instead of indenting.

Reply:
not sure what you mean about textwrap.dedent. found that symbol but had a hard time understanding what it was doing.

Reply:
My point is that a single word command with the same name as a Python standard library command is going to be annoying.   If these become verbs, that is less likely.

Reply:
perhaps just indent/unindent instead of also dedent?